.github/workflows/docker-ci-test.yml

```diff
@@ -0,0 +1,37 @@
+name: push-images-test
+
+on:
+  schedule:
+      - cron: "0 0 */3 * *"
+  workflow_dispatch:
+
+env:
+  REGISTRY: docker.io
+  IMAGE_NAME: initdc/dnsproxy
+
+jobs:
+  buildx:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - name: Set up QEMU
+        uses: docker/setup-qemu-action@v1
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v1
+      - name: Login to DockerHub
+        uses: docker/login-action@v1
+        with:
+          username: ${{ secrets.DOCKER_USERNAME }}
+          password: ${{ secrets.DOCKER_TOKEN }}
+      - name: Extract Docker metadata
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          images: ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}
+      - name: Build and push Docker image
+        run: |
+          set -e
+          docker buildx ls
+          docker buildx build --platform linux/amd64,linux/arm64,linux/riscv64,linux/ppc64le,linux/s390x,linux/386,linux/mips64le,linux/arm/v7,linux/arm/v6 -t initdc/dnsproxy:v$(TZ=Asia/Shanghai date +%Y.%m.%d) . --push
```

Comment:
I wonder... Are Docker hub's times in some TZ or are they plain UTC?

Reply:
I live in time zone UTC+8, It's personal perfer, also an example of time TAG customization

Reply:
OK. IMO, it should be UTC, since it's quite commonly used and understood worldwide. Easier to figure out by others in other TZs.